.scss-lint.yml

```diff
@@ -1,60 +0,0 @@
-# BC configuration
-linters:
-
-  #Cos we use maps with keywords, this produces noise
-  ColorKeyword:
-    enabled: false
-
-  #Cos we now use block comments for documentation
-  Comment:
-    enabled: false
-
-  ElsePlacement:
-    enabled: true
-    style: new_line # or 'same_line'
-
-  EmptyLineBetweenBlocks:
-    enabled: true
-    ignore_single_line_blocks: false
-
-  Indentation:
-    enabled: true
-    character: space # or 'tab'
-    width: 4
```

Comment:
as I see SCSS guidelines uses 2 spaces for Indentation and this leads to lots of changes in files. Does it worth to rewrite it now?